src/components/TableColumnSetup/TableColumnSetup.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
```

Comment:
Why we can't just move the component outside the Table?

Reply:
Let's simply move it up to the components. It should be the one component. Keep backward compatibility with old versions and add sticky feature